### DISCUSSION TITLE
Google Colab notebook update

### DISCUSSION THREAD

**Discussion by HardStepPL** [archinetai/audio-diffusion-pytorch#17] [Q&A] [answered]
Will there still be an updated notebook? I see that there have been no changes for a long time

**Comment by ModeratePrawn** [comment, marked as answer]
Flavio indicates in this issue comment (https://github.com/archinetai/audio-diffusion-pytorch/issues/21#issuecomment-1257241714) that he may publish a new notebook in a week or so.

**Reply by flavioschneider** [reply]
Released here https://github.com/archinetai/audio-diffusion-pytorch/discussions/22
